plugins/manual_maps/ManualMapLoading.ts

```diff
@@ -0,0 +1,214 @@
+import fs from 'fs/promises'
+
+import { Logger } from '../../src/Logger.js'
+import { Client } from '../../src/client/Client.js'
```

Comment:
you arent supposed to be calling client or logger directly inside a plugin. tm.client or tm.log do the job just fine.